lib/api/controller/auth.js

```diff
@@ -188,9 +189,74 @@ class AuthController extends NativeController {
         request.context.token);
     }
 
+    if (global.kuzzle.config.security.supportCookieAuthentication
```

Comment:
Nitpicking: coding standard
```suggestion
    if ( global.kuzzle.config.security.supportCookieAuthentication
```